### PR TITLE
Fix/routes

### PR DESCRIPTION
**Pull request by integricho**
TLDR; This takes librarian up to the point where it complains about missing static assets only (note: no plugins are registered here though)
This PR bulids on the work done in fix/import-errors by:
- Returning back the old route configuration for now just all bundled together
- Fixing a couple additional import paths that was not discovered yet due to routes being not imported before
- Fixes registration of template helpers
- The burden of dropping `supervisor` from `CDFObject` to solve the problem of component registration in hooks have carried over a couple mistakes, namely in some places supervisor was still passed into the object, those cases are fixed here

### CODE REVIEW

File: librarian/routes/lang.py [hunk 1]
```diff
@@ -12,20 +12,15 @@
...
 
 from ..core.contrib.templates.renderer import template
+from ..helpers import lang  # NOQA
 
 
 EXPORTS = {
         'routes': {'required_by':
-                   ['librarian_core.contrib.system.routes.routes']}
+                   ['librarian.core.contrib.system.routes.routes']}
 }
```
Comment (goodbadwolf) [inline librarian/routes/lang.py]: Is this needed anymore ?

Comment (integricho) [inline librarian/routes/lang.py]: You're right,it is not honored anymore here.

Comment (foxbunny) [inline librarian/routes/lang.py]: I think the system routes should b moved to routes like other routes.

Comment (integricho) [inline librarian/routes/lang.py]: Yes that should happen

Comment (foxbunny) [inline librarian/routes/lang.py]: Do it now or after merge?

Comment (integricho) [inline librarian/routes/lang.py]: If you guys agree,we can do it now, I tried to limit the amount of changes to only necessary ones to get it running as agreed. but I'm fine with doing this,sure

Comment (foxbunny) [inline librarian/routes/lang.py]: +1

Comment (goodbadwolf) [inline librarian/routes/lang.py]: It would fall under the purpose of this PR, so can be done now

**Comment by foxbunny** [comment]
Looks good to me.
